### DISCUSSION TITLE
TypeError: Failed to fetch when testing

### DISCUSSION THREAD

**Discussion by ghost** [emailjs-com/help#99] [Q&A]
I have set up all the functions as show in the docs. However, whenever I try and send an email, nothing is sent. I checked the error log and found this 'TypeError: Failed to fetch'. Has anyone else experienced a similar issue and can I get some help kindly?

**Comment by xr0master** [comment]
nodejs?

**Reply by ghost** [reply]
No. I'm just using vanilla js
